test/call.c

```diff
@@ -3307,6 +3312,9 @@ int test_call_sni(void)
 	const char *dp = test_datapath();
 	char s[256];
 
+	/* warnings are expected for negative test cases, so silence them */
+	dbg_init(DBG_ERR, DBG_ANSI);
+
```

Comment:
Please also restore the debug level at the end of the test ...

Reply:
Done.